### PR TITLE
fix: remove strange `Ping`

### PR DESCRIPTION
**Pull request by ulbqb**
## Description

`Ping` function is used in only test.  But it always returns nil which is meaningless. So I removed it. Removing has no problem for spec.

https://github.com/informalsystems/tendermint-rs/blob/main/docs/architecture/adr-011-validator-crate.md

**`Ping` History:**

- https://github.com/tendermint/tendermint/pull/2568/files#diff-21b28fcbc3e3f755a54323769b10510e7c144d521e05d065dac8f8360928baa7R79
- https://github.com/tendermint/tendermint/pull/3351/files#diff-4c9597947b77f66a0436065caefa4420fc94bf63faefe22979db4bbfd9720aaeL131
- https://github.com/tendermint/tendermint/pull/3370/files#diff-f26cb79f69b0f0824a4e09e13ce477efdfc05d9ae1101d6cee5eae8ed09539edR51

It seemed to be left behind after repeated refactorings.

### CODE REVIEW

File: test/kms/bench_test.go [hunk 1]
```diff
@@ -42,17 +42,6 @@ func BenchmarkKMS(b *testing.B) {
 	client, err := privval.NewSignerClient(endpoint, chainID)
 	require.NoError(b, err)
 
-	// ensure connection and warm up
-	b.Run("Ping", func(b *testing.B) {
```
Comment (torao) [inline test/kms/bench_test.go:46]: Is it possible to leave the Ping API benchmark using `endpoint` even if SignerClient.Ping() is removed?

Comment (ulbqb) [reply test/kms/bench_test.go:46]: OK, I will implement `Ping` for benchmark.

**Review by torao — APPROVED** [review]
LGTM

**Review by tnasu — APPROVED** [review]
I don't know why, but I've confirmed to stop using `Ping()` in history. So I agree to remove `Ping`.
* Make `Ping()` and use it for monitoring the connection alive
  * https://github.com/tendermint/tendermint/pull/2568
    * https://github.com/tendermint/tendermint/pull/2568/files#diff-21b28fcbc3e3f755a54323769b10510e7c144d521e05d065dac8f8360928baa7R79
* Refator `IPCVal` to `SocketVal`
  * https://github.com/tendermint/tendermint/pull/2923
    * https://github.com/tendermint/tendermint/pull/2923/files#diff-96c36ede118b77af92b95fa88f48345ba89132e516d588b9cc07d35283418394R150
* Refactor to divide `SocketVal` into `SignerRemote` and  `SignerValidatorEndpoint`
  * https://github.com/tendermint/tendermint/pull/3351
    * https://github.com/tendermint/tendermint/pull/3351/files#diff-2a787d21700c92e9c1b812d4c4c439e471c28ee4369640f9fd399b621d2ba035R142
* Refactor `SignerValidatorEndpoint` to `SignerListenerEndpoint` and stop using `Ping()`
  * https://github.com/tendermint/tendermint/pull/3370
    * https://github.com/tendermint/tendermint/pull/3370/files#diff-b15f92c590eafe71fca1b0b951414e66e25cc8dc307522bf8cbdf99ab198ec1cR188

In addition, the current `Ping()` is a strange implementation since the first version.
https://github.com/tendermint/tendermint/pull/3370/files#diff-f26cb79f69b0f0824a4e09e13ce477efdfc05d9ae1101d6cee5eae8ed09539edR54-R63

**Pull request merged**